### ISSUE TITLE
Tooltip of immovables always visible

### ISSUE THREAD

**Issue by matthiakl** [widelands/widelands#4673]
**Describe the bug**
After clicking on an immovable its tooltip stays active everywhere outside the action window

**Expected behavior**
The tooltip should behave the same as with the action window closed.

**Screenshots**
![Bildschirmfoto vom 2021-02-12 15-22-04](https://user-images.githubusercontent.com/23122583/107786491-45271d00-6d4e-11eb-9497-eab30e47960b.png).
